eCommerce platforms/Adobe Magento/src/code/local/Microsoft/Dfp/etc/adminhtml.xml

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0"?>
+<!--
+/**
+ * Microsoft Dynamics 365 Fraud Protection module for Magento
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * http://opensource.org/licenses/osl-3.0.php
+ *
+ * @category   Microsoft
+ * @package    Dynamics 365 Fraud Protection
+ * @copyright  Copyright (c) Microsoft Corporation. (https://www.microsoft.com)
+ * @license    http://opensource.org/licenses/osl-3.0.php  Open Software License (OSL 3.0)
+ */
+-->
+
+<config>
+  <acl>
+    <resources>
+      <admin>
+        <children>
+          <system>
+            <children>
+              <config>
+                <children>
+                  <dfp translate="title" module="dfp">
+                    <title>DFP Config Section</title>
```

Comment:
super nit: any reason to not just call this `DFP Configuration`

Reply:
Nope!  Changed.